doc/sphinxman/source/dftd3.rst

```diff
@@ -373,6 +373,8 @@ any user-specified dispersion parameters.
 
 .. autofunction:: qcdb.Molecule.run_dftd3
 
+.. autofunction:: qcdb.Molecule.run_dftd4
```

Comment:
When is `autofunction` needed?

Reply:
Basically, whenever in narrative docs users would benefit from seeing the (hopefully better maintained) docstring from the code, throw in the autofunction or autoclass. Also, if some narrative or docstring text mentions a fn or class with 

```
:class:`psi4.core.set_memory`
:func:`~psi4.driver.energy`  # tilde means link looks like "energy", not "psi4.driver.energy"
```

then Sphinx wants to link to it. If it hasn't already been covered by an automodapi, then you need to add the autofunction somewhere as a link target. (Miscellaneous end up in `external_apis.rst`.)